text/0000-fabric-private-chaincode-lite.md

```diff
@@ -594,3 +639,19 @@ Typical signature verifications involve a PKI where the ultimate root of trust i
 Note on Terminology: The current feature naming scheme includes several elements that originated in Intel&reg; SGX.
 Today, terms like "Enclave" are widely accepted and used to refer to a generic TEE, rather than a specific technology.
 The project team currently participates in the new Confidential Computing Consortium which aims to promote standards for deployment of workloads across TEEs/Enclaves, and we intend to align with their terminology as it evolves.
+
+# Appendix
```

Comment:
Above you seem to reference also a section on the use-case in the appendix.   If below is really what is meant and it deals with what FPC Lite does (regardingless of use-case), then it should be rather a reference in the architecture, not in the use-case section?